shared/chat/conversation/messages/text.desktop.js

```diff
@@ -111,12 +112,22 @@ const _avatarStyle = {
   marginRight: globalMargins.tiny,
 }
 
-export default withHandlers({
-  onIconClick: (props: Props) => event => {
-    props.onAction(props.message, event)
-  },
-})(_MessageTextComponent)
-
 const stylesFirstNewMessage = {
   borderTop: `solid 1px ${globalColors.orange}`,
 }
+
+export default compose(
+  shouldUpdate((props: Props, nextProps: Props) => {
```

Comment:
We could move to a more powerful styling thing that hopefully solves some of these perf problems, or even not just expose style overrides everywhere and instead encode that in a prop (which is likely more react-y anyways). So instead of doing like style={marginLeft: 4} we'd have like type='Offset' which would do that internally or something. depends on the context